volmdlr/wires.py

```diff
@@ -4739,8 +4739,8 @@ def sewing_with(self, other_poly3d, x, y, normal, resolution=20):
 
         bbox_self2d, bbox_other2d = self_poly2d.bounding_rectangle(), \
             other_poly2d.bounding_rectangle()
-        position = [abs(value) for value in bbox_self2d] \
-            + [abs(value) for value in bbox_other2d]
+        position = [abs(value) for value in bbox_self2d.items()] \
+            + [abs(value) for value in bbox_other2d.items()]
```

Comment:
`bbox_self2d` and `bbox_other2d` are not dictionaries.

Reply:
Indeed. bbox_self2d and bbox_other2d should receive the bounds of the BoundingRectangle. I had created a method called items() for BoundingRectangle but then I changed the method name to bounds(). Thank you